src/scss/atlas-theme/_atlas.scss

```diff
@@ -1,5 +1,5 @@
 @import "alerts";
-@import "breadcrumbs";
+@import "badges";
```

Comment:
@pat270 is this a mistake? When I try to build, it throws an error because `_badges.scss` doesn't exist.